website_variants_extra/views/templates.xml

```diff
@@ -45,12 +46,125 @@
+                                                  "from_currency": "website.currency_id",
+                                                  "display_currency": "user_id.partner_id.property_product_pricelist.currency_id"
+                                               }'/>
+                                     </li>
+                                     <li class="h3 oe_price_h4 css_editable_mode_hidden">
+                                            <strong class="text-danger oe_price" style="white-space: nowrap;"
+                                                t-field="product.price"
+                                                t-field-options='{
+                                                    "widget": "monetary",
+                                                    "display_currency": "user_id.partner_id.property_product_pricelist.currency_id"
+                                                }'/>
+                                             <span itemprop="price" style="display:none;" t-esc="product.price"/>
+                                             <span itemprop="priceCurrency" style="display:none;" t-esc="user_id.partner_id.property_product_pricelist.currency_id.name"/>
+                                     </li>
+                                     <li>
+                                         <h4 class="css_editable_mode_display" style="display: none;">
+                                             <span t-field="product.lst_price"
+                                                 t-field-options='{
+                                                 "widget": "monetary",
+                                                 "display_currency": "website.pricelist_id.currency_id"
+                                             }'/>
+                                         </h4>
+                                     </li>
+                                </ul>
+                            </div>
+                            <div class="css_quantity input-group oe_website_spinner" style="margin-top: 20px;">
+                                <span class="input-group-addon">
+                                    <a t-attf-href="#" class="mb8 js_add_cart_json">
+                                        <i class="fa fa-minus"></i>
+                                    </a>
+                                </span>
+                                <input type="text" class="js_quantity form-control" data-min="1" name="add_qty" value="1"/>
+                                <span class="input-group-addon">
+                                    <a t-attf-href="#" class="mb8 float_left js_add_cart_json">
+                                        <i class="fa fa-plus"></i>
+                                    </a>
+                                </span>
+                            </div>
+                            <a id="add_to_cart" class="btn btn-warning btn-lg mt16 js_check_product a-submit" href="#">Add to Cart</a>
```

Comment:
Can you remove the class `btn-lg` to make the button fit the size asked by the customer.